mitodl/micromasters
# Added ability to edit education to users page

#### What are the relevant tickets?

closes #392 

#### What's this PR do?

This does a similar thing to #406, we move the actual form components out of `EducationTab` and into `EducationForm`, which we can then drop on the `User` page.

#### Where should the reviewer start?

Review the changes to `EducationTab` and `User`.

#### How should this be manually tested?

Make sure that you can edit all educational entries on both the `User` page and on the `EducationTab`.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
